benchmarks/ALIA/sv-benchmarks/2019.array-cav/array_init_var_plus_ind.sl

```diff
@@ -0,0 +1,32 @@
+(set-logic ALIA)
+
+(synth-inv inv_fun ((a (Array Int Int)) (n Int) (j Int) (i Int) (x Int)))
+
+(declare-primed-var a (Array Int Int))
+(declare-primed-var j Int)
+(declare-primed-var i Int)
+(declare-primed-var n Int)
+(declare-primed-var x Int)
+
+
+(define-fun pre_fun ((a (Array Int Int)) (n Int) (j Int) (i Int) (x Int)) Bool
+  (and (>= n 0) (= i 0) (= j 0)))
+
+(define-fun trans_fun ((a (Array Int Int)) (n Int) (j Int) (i Int) (x Int) (a! (Array Int Int)) (n! Int) (j! Int) (i! Int) (x! Int)) Bool
+  (and (and (< i n) (= x 0))
+       (= n! n)
+       (= j! (+ j 1))
+       (= a! (store a i j))
+  )
+)
+
+(define-fun post_fun ((a (Array Int Int)) (n Int) (j Int) (i Int) (x Int)) Bool
+  (or (and (< i n) (= x 0))
+      (forall ((l Int)) (=> (and (>= l 0) (< l i))
+                            (>= 0 (select a l))))
```

Comment:
Is this the source file: https://github.com/sosy-lab/sv-benchmarks/blob/master/c/array-cav19/array_init_var_plus_ind.c?

- It's checking `k` from `1` to `i`. Your `l` goes from `0` to `i`.
- It's checking `a[k] >= 0`. You're checking `0 >= a[l]`